sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -693,193 +712,12 @@ public virtual AsyncPageable<ConfigurationSetting> GetRevisionsAsync(SettingSele
         public virtual Pageable<ConfigurationSetting> GetRevisions(SettingSelector selector, CancellationToken cancellationToken = default)
         {
             Argument.AssertNotNull(selector, nameof(selector));
-            return PageResponseEnumerator.CreateEnumerable(nextLink => GetRevisionsPage(selector, nextLink, cancellationToken));
-        }
-
-        private Request CreateGetRequest(string key, string label, DateTimeOffset? acceptDateTime, MatchConditions requestOptions)
-        {
-            Argument.AssertNotNullOrEmpty(key, nameof(key));
-
-            Request request = _pipeline.CreateRequest();
-            request.Method = RequestMethod.Get;
-            BuildUriForKvRoute(request.Uri, key, label);
```

Comment:
Can you please clean up all the unused methods?

For example: Now we don't need [these](https://github.com/Azure/azure-sdk-for-net/blob/8ad3e7480fa1ee9695dc114354d57b5af1f07594/sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient_private.cs#L61-L141) methods, so we can remove it.